files/playbooks/quincy/ceph-bootstrap-dashboard.yml

```diff
@@ -9,6 +9,8 @@
     ceph_dashboard_password: password
     ceph_dashboard_port: 7000
     ceph_dashboard_username: admin
+    ceph_dashboard_standby_behaviour: redirect
+    ceph_dashboard_standby_error_status_code: 503
```

Comment:
Mention https://docs.ceph.com/en/quincy/mgr/dashboard/#configure-the-error-status-code here.

Reply:
Note and links added